lib/board.rb

```diff
@@ -0,0 +1,49 @@
+class Board
+
+	attr_accessor :cells, :win_combinations
+
+	def initialize
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.